Uni_life_tracker/app/src/main/java/com/generic/plannr/Database/DatabaseClient.java

```diff
@@ -0,0 +1,45 @@
+package com.generic.plannr.Database;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+
+import androidx.annotation.Nullable;
+
+public class DatabaseClient extends SQLiteOpenHelper {
```

Comment:
It would be nice if this had javadocs for the class and its methods

Reply:
Done

Reply:
I didn't add javadocs for onCreate and onUpgrade since these are overriden